Pages/Index/Index.cshtml

```diff
@@ -1,17 +1,22 @@
+@page "/"
+
+@* only pull in the tag helpers we need *@
+@addTagHelper "Microsoft.AspNetCore.Mvc.TagHelpers.LinkTagHelper, Microsoft.AspNetCore.Mvc.TagHelpers"
```

Comment:
I could do `*` here to enable all tag helpers, but I'm trying to keep this as lean as possible so I'm only enabling the tag helpers I know I need...